README-new.md

```diff
@@ -0,0 +1,4 @@
+HacktoberFest
+Code written during HacktoberFest event. hacktoberfest.digitalocean.com
+
+Hacktoberfest is a yearly one month long virtual festival event to celebrate open source contributions presented by Digital Ocean, Intel and DEV. It encourages people to contribute to open source in October celebrating a sense of community, learning and giving back. It is the easiest way to get into open source! During this event, anyone can support open code by contributing changes and earn limited-edition swag
```

Comment:
Hello Krishna :)
Thank you for submitting this pull request. I saw the changes and it seems you are proposing the creation of a new `README-new.md` file which would contain exactly the same content the existing `README.md` file does contain. I did not understand the purpose of it, would you mind kindly describing its usefulness?
Thank you!